src/main/java/net/sourceforge/jwbf/core/contentRep/WatchListResults.java

```diff
@@ -0,0 +1,71 @@
+package net.sourceforge.jwbf.core.contentRep;
+
+import java.util.List;
+
+import net.sourceforge.jwbf.core.internal.Checked;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonNaming(PropertyNamingStrategy.LowerCaseStrategy.class)
+@JsonInclude(Include.NON_DEFAULT)
+public class WatchListResults {
```

Comment:
looks good, can you move this class and WatchResponse to src/main/java/net/sourceforge/jwbf/mediawiki/actions/queries/ please?